test/lib/mayaUsd/fileio/testAddMayaReference.py

```diff
@@ -40,12 +40,11 @@ class AddMayaReferenceTestCase(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        fixturesUtils.readOnlySetUpClass(__file__, loadPlugin=False)
-        if not cls.pluginsLoaded:
-            cls.pluginsLoaded = mayaUtils.isMayaUsdPluginLoaded()
+        fixturesUtils.setUpClass(__file__)
```

Comment:
No need to create a Maya file in a temporary directory, using the fixture utils sets up a temporary directory for the test, and clears it out before running.  Just use setUpClass() instead of readOnlySetUpClass().  Also (have seen this in a few places) why pass in loadPlugin=False to the fixturesUtils, then load the plugin separately?  Just use the fixturesUtils service.